src/openrct2/world/TileElement.h

```diff
@@ -272,7 +277,7 @@ struct PathElement : TileElementBase
 
     bool IsLevelCrossing(const CoordsXY& coords) const;
 };
-assert_struct_size(PathElement, 16);
+assert_struct_size(PathElement, 17);
```

Comment:
I doubt this will pass. These asserts are here to maintain SV6 compatibility. You can see a number of recent contributions to see how to deal with that: read old structs from files, use them to set new ones.

Reply:
Its not to maintain SV6 compatibility as that was 8 bytes. These asserts are because we don't want this struct to exceed 16 bytes. So its still the same outcome in that we can't make this change.